scripts/prestart.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+# FIXME: this is an interim solution until paritytech/substrate-light-ui#10 gets merged
+# which is currently blocked by facebook/create-react-app#5645
+
+echo Prebuilding Dependent Apps...
+
+# Exit immediately if a command exits with a non-zero status
+trap 'exit' ERR
+
+ROOT=`dirname "$0"`
+PROJECT_ROOT=`git rev-parse --show-toplevel`
+
+# A list of dependent directories to build before light-apps
+SRCS=(
+	"packages/ui-components/"
+  "packages/ui-api/"
+	"packages/identity-app/"
+  "packages/transfer-app/"
```

Comment:
weird indentation

Reply:
mm that's weird, i'm not seeing that on my machine 
![screen shot 2019-02-11 at 12 23 00 pm](https://user-images.githubusercontent.com/10432070/52546134-d11cac00-2df7-11e9-8821-91f19b60236f.png)
![screen shot 2019-02-11 at 12 23 12 pm](https://user-images.githubusercontent.com/10432070/52546137-d37f0600-2df7-11e9-940a-774b2ba8e4c7.png)